readme.txt

```diff
@@ -21,6 +21,12 @@ Sendsmaily newsletter subscription plugin for WordPress.
 
 == Changelog ==
 
+= 2.0.0 =
```

Comment:
Reset the changelog. I mean 2.0.0 version info should be the only entry in changelog and it should also describe the motivation behind making the next major release. Column name change and dbDelta implementation isn't enough, as users would need to know who `dbDelta` works in order to understand the change.

Reply:
Then again. Maybe mentioning previous version isn't such a good idea. Let's just change it to:
```
### 2.0.0

First public release.
```